tests/test_models_variational.py

```diff
@@ -128,8 +128,16 @@ def test_vqe(backend, method, options, compile, filename):
     np.random.seed(0)
     initial_parameters = np.random.uniform(0, 2 * np.pi, 2 * nqubits * layers + nqubits)
     v = models.VQE(circuit, hamiltonian)
+
+    def callback(parameters):
+        pass
```

Comment:
Perhaps here it is better to add something to check if the callback is working properly